test/unit_test/systems/python_environment/test_python_environment_system.py

```diff
@@ -100,7 +100,7 @@ def test_evaluate_accuracy(self, mock_compute_accuracy):
         expected_call = mock_compute_accuracy.mock_calls[0]
         # python environment call
         actual_call = mock_compute_accuracy.mock_calls[1]
-        assert actual_call.args[0] == expected_call.args[0]
```

Comment:
why expected_call.args[0] is different with metric?

Reply:
It is changed with the reason of (`1actual_call.args[0]` is metric config):
1. `actual_call` directly call `compute_accuracy` which won't be generated config with model's io_config.
2. `exptected_call` call `evaluate_model` as the accuracy method for local system is private. But `evaluate_model` will generate a metric config with model's io_config. It made `actual_call.args[0] != expected_call.args[0]`

Reply:
I changed the actual_call with `evalaute_model` to align the tests. 

Also @jambayk  Could you help double check? If we can make sure the `compute_accuracy` won't be call beyond the `evalatue_model`, then it is fine. Otherwise, we may need to update metric io config in the level of `compute_latency/accuracy`

Reply:
Yes, it looks good to me. thanks!